tests/conftest.py

```diff
@@ -2728,6 +2728,12 @@ def ceph_toolbox(request):
     deploy = config.RUN["cli_params"]["deploy"]
     teardown = config.RUN["cli_params"].get("teardown")
     skip_ocs = config.ENV_DATA["skip_ocs_deployment"]
+    # In Openshift dedicated platform, OCS is installed via osde2e tool and
+    # it is not installed via OCS-CI. Thus skip_ocs_deployment is set True.
+    # However during the test execution, Ceph toobox is not deployed when
+    # skip_ocs_deployment is True, so skip_ocs is set False here.
+    if config.ENV_DATA["platform"].lower() == "openshiftdedicated":
+        skip_ocs = False
```

Comment:
```suggestion
    deploy_teradown = deploy or teardown
    ocp_dedicated = config.ENV_DATA["platform"].lower() == "openshiftdedicated"
    if not (deploy_teardown or skip_ocs) or (ocp_dedicated and not deploy_teradown):
```
Which will be more readable that we would like to deploy toolbox pod for ocp_dedicated for test execution and not for teardown/deploy. WDYT?

Reply:
I agree on the readability and fixed accordingly